mkdocs/docs/tutorials/knative/canary-progressive.md

```diff
@@ -203,11 +211,11 @@ Observe the experiment in realtime. Paste commands from the tabs below in separa
 
     As the experiment progresses, you should see traffic progressively shift from `sample-app-v1` to `sample-app-v2`. When the experiment completes, all of the traffic will be sent to the winner, `sample-app-v2`.
         
-## 5. Cleanup
+## 6. Cleanup
 ```shell
 kubectl delete -f $ITER8/samples/knative/canaryprogressive/experiment.yaml
 kubectl delete -f $ITER8/samples/knative/canaryprogressive/fortio.yaml
-helm uninstall sample-app --namespace=iter8-system
+helm uninstall sample-app --namespace=default
 ```
 
 ???+ info "Understanding what happened"
```

Comment:
Understanding what happened also needs to change -- `iter8-system` must become `default`